docs/excel/custom-functions-web-reqs.md

```diff
@@ -162,7 +162,7 @@ CustomFunctions.associate("INC", increment);
 ```
 
 >[!NOTE]
-> Note that there are also a category of functions called cancelable functions, which are *not* related to streaming functions. Previous versions of custom functions required you to declare `"cancelable": true` and `"streaming": true` in JSON written by hand. Since the introduction of autogenerated metadata, only asynchronous custom functions which return one value are cancelable. Cancelable functions allow a web request to be terminated in the middle of a request, using a [`CancelableInvocation`](https://docs.microsoft.com/javascript/api/custom-functions-runtime/customfunctions.cancelableinvocation?view=office-js) to decide what to do upon cancellation. Declare a cancelable function using the tag `@cancelable`.
+> Note that there are also a category of functions called cancelable functions, which are *not* related to streaming functions. Previous versions of custom functions required you to declare `"cancelable": true` and `"streaming": true` in JSON written by hand. Since the introduction of autogenerated metadata, only asynchronous custom functions which return one value are cancelable. Cancelable functions allow a web request to be terminated in the middle of a request, using a [`CancelableInvocation`](/javascript/api/custom-functions-runtime/customfunctions.cancelableinvocation?view=office-js) to decide what to do upon cancellation. Declare a cancelable function using the tag `@cancelable`.
```

Comment:
Can we also remove the link param?

```suggestion
> Note that there are also a category of functions called cancelable functions, which are *not* related to streaming functions. Previous versions of custom functions required you to declare `"cancelable": true` and `"streaming": true` in JSON written by hand. Since the introduction of autogenerated metadata, only asynchronous custom functions which return one value are cancelable. Cancelable functions allow a web request to be terminated in the middle of a request, using a [`CancelableInvocation`](/javascript/api/custom-functions-runtime/customfunctions.cancelableinvocation) to decide what to do upon cancellation. Declare a cancelable function using the tag `@cancelable`.
```